selfdrive/test/process_replay/vision_meta.py

```diff
@@ -0,0 +1,46 @@
+from collections import namedtuple
+from cereal.visionipc import VisionStreamType
+from common.realtime import DT_MDL, DT_DMON
+from common.transformations.camera import tici_f_frame_size, tici_d_frame_size, tici_e_frame_size, eon_f_frame_size, eon_d_frame_size
+
+class CameraFrameSizes:
```

Comment:
this is just a dict?

Reply:
it mimics the behavior of a dict - that way i don't have to list sizes for every possible device type

Reply:
why not just use a dict?

Reply:
it would be much larger and would have duplicate entries for tizi and tici

Reply:
but it's the same thing, i can make separate declarations for each camera type